ttnn/cpp/ttnn/operations/eltwise/unary/device/unary_composite_op.cpp

```diff
@@ -35,6 +33,35 @@ Tensor _tanhshrink(const Tensor& x, const std::optional<MemoryConfig>& output_me
     return result;
 }
 
+// power - floating point exponent
+Tensor _power(uint8_t queue_id, const Tensor& input_a, float exponent, const std::optional<MemoryConfig>& output_mem_config, const std::optional<Tensor>& output_tensor) {
```

Comment:
Previously we always passed memory config as the last parameter. 
Also @ayerofieiev-tt  what was your decision on queue_id?

Reply:
@dmakoviichuk-tt , I do not expect any changes on MCW side at this moment.

- queue_id will be changed to context, queue_id will be moved inside of it. 
- right now optional output tensor typically is last and memory config is before it, **but it should be optional, instead of const ref**.

Reply:
Passing optional by value will make additional copy if tensor is inside but yes in all other places it is passed by value.